end_to_end_test/lib/enums_nnbd.dart

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) 2017, Google Inc. Please see the AUTHORS file for details.
+// All rights reserved. Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+// @dart=2.9
```

Comment:
Out of curiosity -- is this necessary? Wouldn't this get Null Safety by default and you only need to mark the legacy parts as `2.8`?

Reply:
Yes, although the 2.9 means that if it's not using the new version for some reason we get an error, which is handy.

Neither option seems perfect though :)